setup.py

```diff
@@ -32,6 +32,21 @@ def read_version():
     return read("VERSION").strip()
 
 
+def pysdk_version():
```

Comment:
This seems a little strange to include in the setup.py. Is this the best way to get the PySDK version?

Reply:
I am still looking for a better way. Any suggestion here?

Reply:
@gwang111 and @makungaj1 We don't need to create a function to just get SDK version. We can do something like -
```python
import importlib_metadata

SDK_VERSION=importlib_metadata.version("sagemaker")
```

Reply:
Tagging @mohanasudhan to check if he approves to this.

Reply:
You could try this:
```
SDK_VERSION = importlib_metadata.version("sagemaker")
```

https://github.com/aws/sagemaker-python-sdk/blob/87d661a8caff9c0671bce183000be6133d9103b9/src/sagemaker/user_agent.py#L21

Reply:
Curious, why do we need a function to pull the version? What is the problem with above listed option?